src/pages/Sessions.tsx

```diff
@@ -201,8 +256,34 @@ const Sessions = () => {
         </Box>
         {selectedSession && (
           <>
+            <FormControl variant="outlined">
+              <InputLabel id="view">View</InputLabel>
+              <Select
+                id="select"
+                label="view"
+                labelId="view"
+                value={!attendanceView ? "default" : "attendance"}
+              >
+                <MenuItem
+                  value="default"
+                  onClick={() => setAttendanceView(false)}
+                >
+                  Default view
+                </MenuItem>
+                <MenuItem
+                  value="attendance"
+                  onClick={() => setAttendanceView(true)}
+                >
+                  Attendance view
+                </MenuItem>
+              </Select>
+            </FormControl>
```

Comment:
i'm not sure we want to display this select on the "all classes" page since users can't take attendance there. can we render this only if we're not on the all classes tab?